### DISCUSSION TITLE
Toolow, a tool for raising the volume on video files

### DISCUSSION THREAD

**Discussion by Inveracity** [wailsapp/wails#2056] [Show and tell]
Heya everyone!

I used Wails 2.1.0 to write an app for normalizing audio on video files. It does this by wrapping `ffmpeg` commands.

![](https://i.imgur.com/gIhwTro.gif)

source: https://github.com/Inveracity/toolow

Thanks for making Wails, it's awesome to work with 🎉

**Comment by leaanthony** [comment]
Nice work! 👍
